src/pbt/v2/client/airflow_client/composer_client.py

```diff
@@ -0,0 +1,183 @@
+import os
+import re
+import tempfile
+from abc import ABC
+from pathlib import Path
+from typing import Optional
+
+from google.cloud import secretmanager, storage
+from google.cloud.storage import Blob
+from google.oauth2 import service_account
+from google.oauth2.gdch_credentials import ServiceAccountCredentials
+from requests import HTTPError
+
+from src.pbt.v2.client.airflow_client.mwaa_client import retry
+from src.pbt.v2.exceptions import DagUploadFailedException, DagFileDeletionFailedException
+from src.pbt.v2.project_models import DAG
```

Comment:
1. we should be using relative imports
2. src should not be made into a module

Reply:
looks like python default import manager always imports from the full path. 
Will fix it.